tests/main/interfaces-desktop/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Ensure that the desktop interface works.
+
+details: |
+    The desktop interface allows to access the different resources.
+
+    The test-snapd-desktop snap checks files and dirs are accessible through the
+    desktop interface.
+
+systems: [-ubuntu-core-*]
+
+prepare: |
+    echo "Given the desktop snap is installed"
+    snap try $TESTSLIB/snaps/test-snapd-desktop
+
+execute: |
+    CONNECTED_PATTERN=":desktop +test-snapd-desktop"
+    DISCONNECTED_PATTERN="\- +test-snapd-desktop:desktop"
+
+    dirs="/var/cache/fontconfig /usr/share/icons /usr/share/pixmaps"
+    files="/etc/xdg/user-dirs.conf /etc/xdg/user-dirs.defaults"
+
+    echo "The plug is connected by default"
+    snap interfaces | MATCH "$CONNECTED_PATTERN"
+
+    echo "Then the snap is able to desktop files and directories"
```

Comment:
s/able to/able to access/

Reply:
This is coming from a patch by @sergiocazzolato